frontend/src/locales/es/payment.json

```diff
@@ -14,6 +14,6 @@
     "close": "All Good, back to merchant",
     "invalid_payment": "Invalid payment request.",
     "processing": "Processing",
-    "error_occured": "An error occured"
+    "error_occured": "An error occured. Please contact support."
```

Comment:
Yinon asked for this message?

Reply:
No, I added it the entire 'error_occured" key. If we need a specific message let me know and I'll adjust it.